libbeat/tests/system/config/mockbeat.yml.j2

```diff
@@ -98,8 +98,8 @@ setup.ilm:
 
 #================================ Logging =====================================
 
-{% if metrics_period -%}
-logging.metrics.period: {{ metrics_period }}
+{% if file_name -%}
+logging.files.name: {{ file_name }}
```

Comment:
The export command being tested disables environment variable resolution.
https://github.com/elastic/beats/blob/4727470f7f7c3fd31c4b3c9f82c77beb8fe9b070/libbeat/cmd/export/config.go#L42
I think it's so `export config` can be used to copy the config and ignore the local environment.
Originally the test was for `logging.metrics.period` but seeing as how this pr adds the definition for this attribute as a `time.Duration` the export fails because the string could not be parsed as a duration.
I instead changed the test to use a string type attribute so the export does not fail.

However going forward the `export config` command will run into this issue if `logging.metrics.period` or `logging.metrics.enabled` are set to environment variables.
This should not effect other commands (which resolve environment variables).

tldr; attribute+test has changed because environment variable expansion is disabled for the `export` commands